backend/server/router/question_router.py

```diff
@@ -185,3 +186,21 @@ async def delete_all_submissions_from_db():
         return response.json()
     except Exception as e:
         return e
+    
+@router.put("/history/accepted")
+async def update_question_status_to_accepted(userID: str, titleSlug: str):
+    try:
+        status = QUESTION_STATUS.ACCEPTED
+        response = requests.put(config.question_service_url + f"/history?userID={userID}&titleSlug={titleSlug}&status={status}")
+        return response.json()
+    except Exception as e:
+        return e
+    
+@router.put("/history/attempted")
+async def update_question_status_to_attempted(userID: str, titleSlug: str):
```

Comment:
I think u can combine these 2 routes to 1 since its the same function, the status can be the param/body